### ISSUE TITLE
Cannot start any builds / pipelines show blank page

### ISSUE THREAD

**Issue by eivantsov** [openshiftio/openshift.io#232]
Freshly registered, created a space, connected my GitHub account. 

It does not matter if I use a starter sample or import own codebase, everything related to builds is inaccessible for me:

![image](https://user-images.githubusercontent.com/5337267/27173475-807a7fca-51c1-11e7-86c8-7ff8515719e5.png)

Any clues on what i am doing wrong?

**Comment by joshuawilson** [comment]
@eivantsov we currently only have one working path. You can Add to a Space either during creation of the space or from the Analyze tab. In there you should pick the button on the right "Create a new Quick Start project". You will need a unique name for the project but otherwise you can just click finish. You will need to build it with the default vert.x quickstart.
All the other paths through the system are a work in progress.

**Comment by eivantsov** [comment]
@joshuawilson the problem is that as soon as I create a space, I get an error that somth is wrong with pipelines, so I did not even have time to mess it up :) :

![image](https://user-images.githubusercontent.com/5337267/27214366-ba9c3ac8-5274-11e7-968d-6ad1e2a934d7.png)

**Comment by jstrachan** [comment]
it looks like your OpenShift token is invalid. I wonder if you need to relink your OpenShift account?

**Comment by rawlingsj** [comment]
One other thing to check is the namespace (openshift project) names you have.  I think the tenant init service assumes that you already have a namespace that matches your username.  I've seen cases where existing OSO users have a different 'home' namespace which doesn't match their userid.

If relinking your openshift account as @jstrachan mentioned above you could check what namespaces you have.  i.e. `oc get projects`

**Comment by eivantsov** [comment]
@jstrachan just spent 10 mins finding out how to unlink my OpenShift account - all in vain. Profile and settings do not have anything like that. Tried docs link on the getting started page - https://access.redhat.com/documentation/en-us/red_hat_openshift.io/1/html-single/user_guide - and it's 404 for me.

After I have deleted all the auto generated pods on OpenShift Online side (they were then recreated), pipelines worked for me (kind of - the build is stuck -  `Still waiting to schedule task; jenkins-slave-q2bnw-3blcx is offline`). But that's a different issue.
